src/hooks/useBaseBreadcrumbs.tsx

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Svg } from 'components/Svg';
+import { auth, account, Breadcrumb } from 'redux/modules';
+import { page } from 'resources';
+import { getAppletData } from 'utils/getAppletData';
+
+export const useBaseBreadcrumbs = () => {
+  const { id } = useParams();
+  const authData = auth.useData();
+  const appletsFoldersData = account.useFoldersApplets();
+
+  const [baseBreadcrumbs, setBaseBreadcrumbs] = useState<Breadcrumb[]>([]);
+
+  useEffect(() => {
+    let authCrumbs: Breadcrumb[] = [];
+    let appletCrumbs: Breadcrumb[] = [];
+
```

Comment:
I would simplify this const baseBreadcrumbs =[]
baseBreadcrumbs.push({})
baseBreadcrumbs.push({})
setBaseBreadcrumbs(baseBreadcrumbs)

Reply:
done